engine/src/main/java/org/terasology/engine/logic/inventory/ItemPickupAuthoritySystem.java

```diff
@@ -82,7 +82,7 @@ public void updateExtentsOnBlockItemBoxShape(OnAddedComponent event, EntityRef i
         BlockFamily blockFamily = blockItemComponent.blockFamily;
 
         if (blockFamily == null) {
-            LOGGER.warn("Prefab {} does not have a block family", itemEntity.getParentPrefab().getName());
+            LOGGER.atWarn().addArgument(() -> itemEntity.getParentPrefab().getName()).log("Prefab {} does not have a block family");
```

Comment:
low impact and only called in rare case that prefab does not have a block family
```suggestion
            LOGGER.warn("Prefab {} does not have a block family", itemEntity.getParentPrefab().getName()); //NOPMD
```